feat: add snapshot and governance forum links

0xAeterno opened the pull request:
## Related Ticket

<!-- If your changes are related to a github ticket, please provide the issue number: -->

Resolves [#125](https://github.com/KlimaDAO/klimadao/issues/125)

Changes in lieu of the designer's approval

| Screen (Theme) |  Before |  After  |  
| ---------------- | ------- | ------ |
|  Desktop (Light) |  ![image](https://user-images.githubusercontent.com/97446324/153617195-64f931a7-a50d-424c-8396-3c2d4ccdf041.png)  |  ![image](https://user-images.githubusercontent.com/97446324/153616781-cbcb1b1e-efdb-4195-bd6c-da7034e8125b.png) |
|  Desktop (Dark) |  ![image](https://user-images.githubusercontent.com/97446324/153617135-e73c371f-4b50-450e-ab7f-3dbaa4c25231.png)  |  ![image](https://user-images.githubusercontent.com/97446324/153616847-4f133ffd-6672-48f5-9eaa-1359ebdbe6ae.png) |
| Mobile (Light)|![image](https://user-images.githubusercontent.com/97446324/153699049-9f099937-16b0-4819-a08a-b26ba11cb772.png)|![image](https://user-images.githubusercontent.com/97446324/153699041-26923a7c-bb8a-48ac-a40c-734ad3fc6e1a.png)|
| Mobile (Dark)|![image](https://user-images.githubusercontent.com/97446324/153699020-21533e94-0b99-44c1-971f-cb4a966c9ade.png)| ![image](https://user-images.githubusercontent.com/97446324/153698999-fbd3326f-00ff-49e8-add8-a0728bd80fd1.png)|



### Other changes
| Change | Before | After |
| ------- | ------- | ------ |
| Added extra margin around the discord button| ![image](https://user-images.githubusercontent.com/97446324/153617477-7a074079-5e0c-4eda-913c-8695d8f31c88.png) | ![image](https://user-images.githubusercontent.com/97446324/153617697-87fd11ed-da8c-4e6d-9cd7-a33fe981dd77.png) |
| Fixed mobile resource nav bar hover state |![image](https://user-images.githubusercontent.com/97446324/153713345-20512a0d-901f-454c-beed-dd66595cd06f.png)| ![image](https://user-images.githubusercontent.com/97446324/153716785-0597d8b0-f15d-4523-a55d-25eb2fd2537c.png)|


### Discussion points/ Todos
- [x] Links to governance forums and snapshot on smaller screen formats
- [x] Fix light theme mobile resource header hover state
## Checklist

<!-- Check completed item: [X] -->

- [x] Building the site with `npm run build-site` works without errors
- [x] Building the app with `npm run build-app` works without errors
- [x] I formatted JS and TS files with running `npm run format-all`

Atmosfearful reviewed (changes requested):
NICE!!! I spoke with designer and he likes it :) 2 changes please:
* [ ] You should be able to achieve all the spacing with just flex, flex-gap, flex-wrap. And if needed, vertical spacing with grid & row-gap. We avoid margin especially `margin-right` and `margin-left` because it will break when we add RTL languages. https://github.com/KlimaDAO/klimadao/wiki#margin-considered-harmful
* [ ] Let's do something like this for mobile views. This means moving the mobile resources nav up above the textgroup

![image](https://user-images.githubusercontent.com/88635679/153637536-01115c02-e2d7-4bf9-bdb1-7d82f3c1db5f.png)

0xAeterno commented:
> NICE!!! I spoke with designer and he likes it :) 2 changes please:
> 
> * [ ]  You should be able to achieve all the spacing with just flex, flex-gap, flex-wrap. And if needed, vertical spacing with grid & row-gap. We avoid margin especially `margin-right` and `margin-left` because it will break when we add RTL languages. https://github.com/KlimaDAO/klimadao/wiki#margin-considered-harmful
> * [ ]  Let's do something like this for mobile views. This means moving the mobile resources nav up above the textgroup
> 

Thanks for the feedback!
The PR has been updated to favour use of gap over margin. Will keep this in mind for future contributions 😁 
Have also moved the mobile nav above the text group and updated the screenshots in the description

Atmosfearful reviewed (approved):
🏅🏅🏅
